packages/frontend/components/composed/issue/IssueHero/IssueHero.tsx

```diff
@@ -69,10 +69,16 @@ export default function IssueHero({ data }: Props) {
               </Styled.Issue>
             </Styled.IssueText>
             <Styled.IssueMeta>
+              {issue.issn && (
+                <Styled.MetaItem className="t-label-sm">
+                  ISSN: <Styled.Number>{issue.issn}</Styled.Number>
+                </Styled.MetaItem>
+              )}
               {issue.doi && (
-                <span className="t-label-sm">
-                  DOI: <Styled.Number>{issue.doi}</Styled.Number>
-                </span>
+                <Styled.MetaItem className="t-label-sm">
```

Comment:
nit: if this should be have as a block (see item styling), then I suggest replacing this Styled.MetaItem with a plain div.